python/pyspark/sql/pandas/group_ops.py

```diff
@@ -636,8 +630,6 @@ def applyInArrow(
         into memory, so the user should be aware of the potential OOM risk if data is skewed
         and certain groups are too large to fit in memory.
 
-        This API is unstable, and for developers.
```

Comment:
Let's keep this for `...InArrow` ones

Reply:
`mapInArrow` should be stable right? This applyInArrow is indeed a new API.